app/views/submissions/browse.html.erb

```diff
@@ -7,6 +7,7 @@
           show_grace_tokens: <%= @assignment.submission_rule.type == 'GracePeriodSubmissionRule' %>,
           show_sections: <%= Section.exists? %>,
           is_admin: <%= @current_user.admin? %>,
+          is_ta: <%= @current_user.ta? %>,
```

Comment:
A user can't be both a grader and an admin so it might be better to combine the `is_admin` and `is_ta` props somehow.